internal/wal/wal_sqlite.go

```diff
@@ -1,150 +0,0 @@
-// Copyright (c) 2022-present, DiceDB contributors
-// All rights reserved. Licensed under the BSD 3-Clause License. See LICENSE file in the project root for full license information.
-
-package wal
-
-import (
-	"database/sql"
-	"fmt"
```

Comment:
If we are not supporting SQLite as WAL engine, update the `Config` struct to reflect the same. There in possible values there was a reference to `sqlite`. Also, do run `go mod tidy` to eradicate any stray dependency.